Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```diff
@@ -67,6 +67,7 @@
     node: filingCabinet
   - type: StaticPrice
     price: 20
+  - type: RequireProjectileTarget
```

Comment:
Filing cabinets, or at least the tall cabinets, should probably be able to be used as cover.